src/storage/config.rs

```diff
@@ -61,3 +61,50 @@ impl Config {
         Ok(())
     }
 }
+
+#[cfg(test)]
+pub mod tests {
+    use super::*;
+
+    extern crate serde_test;
+    use self::serde_test::{assert_de_tokens, Token};
+
+    pub fn default_de_tokens() -> Vec<Token> {
+        let config = Config::default();
+        vec![
+            Token::Struct {
+                name: "Config",
+                len: 7,
+            },
+
+            Token::Str("data_dir"),
+            Token::Str(DEFAULT_DATA_DIR),
+
+            Token::Str("gc_ratio_threshold"),
+            Token::F64(config.gc_ratio_threshold),
+
+            Token::Str("scheduler_notify_capacity"),
```

Comment:
I think it should be `scheduler-notify-capacity`. It seems that the case is useless.